### PR TITLE
Add internaldevtools command to open developer tools of main window.

### PR DESCRIPTION
**Pull request by gebner**
It's very useful to be able to quickly open the devtools for a running vieb instance, e.g. to figure out why some action takes longer than expected.

I didn't think too hard about the command name.  It might be nice to open the internal dev tools as a tab, but this would probably require larger changes to addTab().

This already came up in another issue: https://github.com/Jelmerro/Vieb/issues/90#issuecomment-727245720 https://github.com/Jelmerro/Vieb/issues/90#issuecomment-727229717

### CODE REVIEW

File: app/pages/help.html [hunk 1]
```diff
@@ -465,6 +465,11 @@ <h3 id=":devtools">:devtools</h3>
             <li><span class="command-block">:devtools tab</span> - Open the developer tools of the current page in a new tab</li>
         </ul>
         The devtools command can be used to open the developer tools of the current page. If no argument is given for the devtools position, it will open at the position configured by <a href="#devtoolsposition">devtoolsposition</a>. The accepted positions for this command are the same as that of the setting. It's not possible to open multiple instances of the devtools for the same page, you can open one type of devtools per tab. By default these tools will have dark mode and some security settings enabled. If the devtools are opened as a Vieb tab or split (so not windowed) all regular Vieb action can be used to interact with the developer tools.
+        <h3 id=":internaldevtools">:internaldevtools</h3>
+        <ul>
+            <li><span class="command-block">:devtools</span> - Open the developer tools of the Vieb main browser window</li>
```
Comment (Jelmerro) [inline app/pages/help.html:470]: The example command is incorrect.

**Comment by Jelmerro** [comment]
I expect that this was only tested with mouse enabled. Because if I use the `--debug` option, I can close the devtools again with `Ctrl-Shift-I`, but that doesn't work when using this command (without `--debug`). I am not too familiar with the devtools shortcuts myself, so I would expect Vieb's help docs to include a way to close them. Alternatively, there could be an option to switch the focus between the devtools and the Vieb. Or finally, we could force the devtools to open in a new window instead of in this embedded position (not sure if that's default or not, but that will not work correctly if not opened with `--debug`). This last option could actually make `--debug` simpler as well.

We could specify the [position to open the tools via the "mode" property](https://www.electronjs.org/docs/api/web-contents#contentsopendevtoolsoptions). I would suggest either "detach" or "undocked". With undocked we keep the option for users to move it back to within the Vieb window. Though there seems to be no way to close them using the keyboard if moved back to the same window (you either need to use the mouse in that case, or undock the tools again). We could actually change the default postition for the `--debug` tools to "undocked" as well, so we won't need the window frame at all.

In short, I want to keep using the keyboard to open or close the devtools, and undocking them by default seems to be the simplest way to achieve this, without requiring the window frame to be there. Please let me know your opinion on this suggestion.

**Comment by gebner** [comment]
I don't have a strong opinion on the devtools placement, so I've set the `undocked` mode now.  I didn't really notice this because electron will remember the choice if you leave the mode field empty (and the first thing I did when testing was to undock it).

**Review by Jelmerro — APPROVED** [review]
